autopush/db.py

```diff
@@ -523,6 +523,7 @@ def all_channels(self, uaid):
         # Note: This only returns the chids associated with the UAID.
         # Functions that call store_message() would be required to
         # update that list as well using register_channel()
+        # TODO: once expiry is properly integrated, use FilterExpression
```

Comment:
What does that mean?

Reply:
Ideally, we want to use FilterExpression in our calls to DynamoDB. Unfortunately, while really efficient, it doesn't handle our need for backwards compatibility. We have some records that don't have an expiry yet set, so those will not be fetched.

Once the expiry has been put in place for several months, and we can safely assume most current records have it, then we can remove the code I've put in place to handle records that do not have an expiry set.

Reply:
Ah, perhaps filing an issue for that now, and linking it here would help. We could then mark the issue blocked so we can remember when we should remove it.

Reply:
👍 https://github.com/mozilla-services/autopush/issues/1073

Reply:
Oh, I meant referring to the issue in the code itself.